core/urls.py

```diff
@@ -15,5 +15,5 @@
     url(r'^user_search/$', UserSearchView.as_view(), name='user_search'),
 
     # Checks on conditional fields
-    url(r'^check_requires/$', check_requires, name='check_requires'),
+    url(r'^check_requires/$', check_requires, name='check_requires')
```

Comment:
Deze comma moet er eigenlijk weer terug komen. 

We gebruiken overal trailing comma's, om git diffs duidelijk te houden

Reply:
Logisch!